src/element/textElement.ts

```diff
@@ -474,3 +478,7 @@ export const getContainerElement = (
   }
   return null;
 };
+
+export const getContainerDims = (element: ExcalidrawElement) => {
+  return { width: element.width, height: element.height };
+};
```

Comment:
Do you anticipate this getting more complex for some shapes? Otherwise why not just read the width/height from the container directly?